lab3react/src/App.js

```diff
@@ -0,0 +1,109 @@
+import logo from './logo.svg';
+import './App.css';
+import {useState} from "react";
+import TaskList from './TaskList.js';
+import Top from './Top.js';
+import Bottom from './Bottom.js';
+import {useDocument} from "react-firebase-hooks/firestore";
+
+
+function App(props) {
+
+    let initial = true;
+
+    if(props.currentList === "wow"){
+        return (<div className="App">
+            {props.loading && <h1>Loading</h1>}
```

Comment:
You repeat this for every possible return, so don't repeat it. Always add it and have your if/else if chain deeper in the hierarchy to modify the things that are different.